float32 shoulder_rotatation
float32 shoulder_angle
float32 elbow_angle
float32 wrist_rotation
float32 wrist_angle
float32 claw_closure